LandSandBoat/server
# [AMK][Lua] Added check and reminder text to AMK 6, enabled repeatability

**_I affirm:_**

## What does this pull request do?


- added check to make sure player has at least one orb (cardian flavor orbs) before initiating cardian fight in amk mission 6
- added flavor text when on the mission but player has no orbs
- enabled ability to repeat the mission after completing it
- found an off-by-one error in the zone text ids

## Steps to test these changes
`!addmission 10 5`
`!gotoid 17572302`
Touch QM, it should tell player that an orb could cause something to happen
`!addkeyitem orb_of_swords`
Touch QM again, it should start the fight prompt
`!addmission 10 6`
Touch QM again, it should still start the fight prompt if player has an orb and doesn't have the ripe_starfruit KI
